stdlib/public/SDK/Foundation/Calendar.swift

```diff
@@ -90,7 +79,7 @@ public struct Calendar : Hashable, Equatable, ReferenceConvertible, _MutableBoxi
     ///
     /// This calendar does not track changes that the user makes to their preferences.
     public static var current : Calendar {
-        return Calendar(adoptingReference: __NSCalendarCurrent(), autoupdating: false)
+        return Calendar(adoptingReference: __NSCalendarCurrent() as! NSCalendar, autoupdating: false)
```

Comment:
Does `NSCalendar.current` do the wrong thing? I mean, I know we'd rather have that return the unbridged type, but meanwhile…